docs/user-guide/mobile-sdk.md

```diff
@@ -0,0 +1,37 @@
+All SDK libraries available as pre-built packages on the GitHub [release page](https://github.com/mysteriumnetwork/node/releases).
+It can be downloaded and installed manually with any preferred way.
+
+But we also support several standard ways for installing SDK using dependency managers.
+
+## iOS
+
+It is possible to get iOS SDK library using [Carthage](https://github.com/Carthage/Carthage) dependency manager.
+
+You can follow the [Carthage quick start guide] to get familiar how to use it or just use the following steps:
```

Comment:
should we add a link here?

Reply:
You can follow the [Carthage quick start guide] to get familiar with the tool or perform following steps: